src/components/workspace-utils.js

```diff
@@ -39,10 +39,16 @@ export const useWorkspaces = (fieldsArg, stringAttributeMaxLength) => {
     const ws = await Ajax(signal).Workspaces.list(fields, stringAttributeMaxLength);
     workspacesStore.set(ws);
   });
+
+  const refreshSilently = _.flow(withErrorReporting('Error loading workspace list'))(async () => {
+    const ws = await Ajax(signal).Workspaces.list(fields, stringAttributeMaxLength);
+    workspacesStore.set(ws);
+  });
+
   useOnMount(() => {
     refresh();
   });
-  return { workspaces, refresh, loading };
+  return { workspaces, refresh, refreshSilently, loading };
```

Comment:
Adding a separate method here is leaking the workspaces list concerns into the hook. The spinner is shown by the workspaces list. The hook shouldn't be affected by that.